docs/getting-started-objc-macos.md

```diff
@@ -0,0 +1,151 @@
+
+Follow the next steps:
+
+1. Delete **MyClass.cs** file and **Properties** folder.
+
+2. Right click on *Weather project > Add > New File.*
+
+3. Select *Empty Class* and use **XAMWeatherFetcher** as Name and click New.
+
+4. Replace *XAMWeatherFetcher.cs* contents with the following code:
+
+```csharp
+using System;
+using System.Json;
+using System.Net;
+
+public class XAMWeatherFetcher {
+
+	static string urlTemplate = @"https://query.yahooapis.com/v1/public/yql?q=select%20item.condition%20from%20weather.forecast%20where%20woeid%20in%20(select%20woeid%20from%20geo.places(1)%20where%20text%3D%22{0}%2C%20{1}%22)&format=json&env=store%3A%2F%2Fdatatables.org%2Falltableswithkeys";
+	public string City { get; private set; }
+	public string State { get; private set; }
+
+	public XAMWeatherFetcher (string city, string state)
+	{
+		City = city;
+		State = state;
+	}
+
+	public XAMWeatherResult GetWeather ()
+	{
+		try {
+			using (var wc = new WebClient ()) {
+				var url = string.Format (urlTemplate, City, State);
+				var str = wc.DownloadString (url);
+				var json = JsonValue.Parse (str)["query"]["results"]["channel"]["item"]["condition"];
+				var result = new XAMWeatherResult (json["temp"], json["text"]);
+				return result;
+			}
+		}
+		catch (Exception ex) {
+			// Log some of the exception messages
+			Console.WriteLine (ex.Message);
+			Console.WriteLine (ex.InnerException?.Message);
+			Console.WriteLine (ex.InnerException?.InnerException?.Message);
+
+			return null;
+		}
+
+	}
+}
+
+public class XAMWeatherResult {
+	public string Temp { get; private set; }
+	public string Text { get; private set; }
+
+	public XAMWeatherResult (string temp, string text)
+	{
+		Temp = temp;
+		Text = text;
+	}
+}
+```
+
+You will notice that `Using System.Json;` is giving an error, in order to fix this we need to do the following:
+
+1. Double click on **References** folder.
+
+2. Click on **Packages** tab.
+
+3. Check **System.Json**.
+
+4. Click **Ok** button.
+
+Once the above is done all we need to do is build our .NET Assembly by clicking on *Build menu > Build All*  or ⌘+ b. Build Succesful message should appear in the top status bar.
+
+Now right click on *Weather* project node and select *Reveal in Finder,* now in finder go to *bin/Debug* folder, inside it you will find **Weather.dll.**
+
+## Using Embeddinator-4000
+
+If you installed Embeddinator-4000 using our pkg installer you should find **objcgen** in the following path `/Library/Frameworks/Xamarin.Embeddinator-4000/Commands/objcgen` , **objcgen** is the tool that we need to generate a native library from a .NET Assembly.
+
+Open terminal and `cd` into the folder containing Weather.dll and execute the following command:
```

Comment:
Open terminal, `cd` into the folder containing **Weather.dll**, and execute the following command: